src/chrome/content/rules/Ubuntu-FR.org-falsemixed.xml

```diff
@@ -1,19 +1,10 @@
-
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://tour.ubuntu-fr.org/ => https://tour.ubuntu-fr.org/: (60, 'SSL certificate problem: certificate has expired')
 
 	For rules not causing false/broken MCB, see Ubuntu-FR.org.xml.
 
 -->
-<ruleset name="Ubuntu-FR.org (false MCB)" platform="mixedcontent" default_off='failed ruleset test'>
-
-	<!--	Direct rewrites:
-				-->
+<ruleset name="Ubuntu-FR.org (false MCB)" platform="mixedcontent" >
```

Comment:
mixed content issues have been solved, so you can merge this into the main ruleset.

Reply:
@J0WI Done